src/main/java/org/commonjava/util/sidecar/config/ProxyConfiguration.java

```diff
@@ -88,6 +93,7 @@ void init()
     {
         load( true );
         logger.info( "Proxy config, {}", this );
+        bus.publish( STARTUP_INIT, sidecarConfig.localRepository.orElse( DEFAULT_REPO_PATH ) );
```

Comment:
I suggest we don't use event bus to send the historicalContentMap construction logic. I didn't see tracking report should relate with proxy config init. and it's async which is not quite suitable for an init map. You could just construct the map in the PostConstruct method of ReportService instead, which is enough.